core/types/block.go

```diff
@@ -130,26 +130,29 @@ func (r *Randomness) EncodeRLP(w io.Writer) error {
 }
 
 type EpochSnarkData struct {
+	Bitmap    *big.Int
 	Signature []byte
 }
 
 func (r *EpochSnarkData) Size() common.StorageSize {
-	return common.StorageSize(blscrypto.SIGNATUREBYTES)
+	return common.StorageSize(blscrypto.SIGNATUREBYTES + (r.Bitmap.BitLen() / 8))
```

Comment:
nit:  Consider adding a comment stating that this function returns an estimated size.

Reply:
Done!